packages/p2p/src/components/advertiser-page/advertiser-page-profile/__tests__/advertiser-page-profile.spec.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { isMobile } from '@deriv/shared';
+import { render, screen } from '@testing-library/react';
+import { useStores } from 'Stores/index';
+import AdvertiserPageProfile from '../advertiser-page-profile';
+
+const mock_store: DeepPartial<ReturnType<typeof useStores>> = {
+    advertiser_page_store: {
+        advertiser_details_id: 'id1',
+        advertiser_details_name: 'test name',
+        counterparty_advertiser_info: {
+            basic_verification: 1,
+            buy_orders_count: 0,
+            created_time: 1686539211,
+            first_name: '',
+            full_verification: 0,
+            is_online: 0,
+            last_online_time: 1686656506,
+            last_name: '',
+            sell_orders_count: 0,
+        },
+    },
+    general_store: {
+        advertiser_id: 'id2',
+        advertiser_info: {
+            basic_verification: 1,
+            buy_orders_count: 5,
+            created_time: 1686539217,
+            first_name: 'test2',
+            full_verification: 1,
+            is_online: 1,
+            last_online_time: 1686656509,
+            last_name: 'user2',
+            sell_orders_count: 5,
+        },
+    },
```

Comment:
i see that these values are repeated in a few test cases, do you think we can also put these values into a constants file?

Reply:
the fields are actually not repeated. general store and advertiser page store is used at 2- 3places, but the fields used are different.

Reply:
ah ok 👍